src/main/java/com/dev/cinema/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.dev.cinema.service.impl;
+
+import com.dev.cinema.dao.OrderDao;
+import com.dev.cinema.lib.Inject;
+import com.dev.cinema.lib.Service;
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.OrderService;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+
+    @Override
+    public Order completeOrder(List<Ticket> tickets, User user) {
+        Order order = new Order(tickets, user, LocalDateTime.now());
+        return add(order);
```

Comment:
What should happen to the shopping cart's tickets?

Reply:
fixed